app/hooks/useQuestionStateInUrl.ts

```diff
@@ -76,11 +76,6 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
     }
   }, [transition.location])
 
-  useEffect(() => {
-    const suffix = stateString ? ` - ${stateString}` : ''
-    document.title = minLogo ? 'AI Safety FAQ' : `Stampy ${suffix}`
```

Comment:
this was always overriding the page title, now it will only change if the contents change